actions/next-semver/action.yml

```diff
@@ -0,0 +1,16 @@
+name: 'next-semver'
+description: 'Establish the correct semantic version from source and previous release value'
+author: 'GOV.UK Pay'
+inputs:
+  package_version:
+    description: 'version in the project source (pom.xml, package.json etc)'
+    required: true
+  release_version:
+    description: 'version of latest github release'
+    required: true
+outputs:
+  version:
+    description: 'semantic version value of the next release'
+runs:
+  using: 'node16'
```

Comment:
Why not node 18? It's node 18 [here](https://github.com/alphagov/pay-ci/pull/823/files#diff-69f6e913dda2dfe289d5b99cb0273ae4fb25629be87ec94285347c241ad867f4R17).

Reply:
those are the type definitions needed for typescript, not runtime-specific. See https://github.com/alphagov/pay-ci/pull/823#discussion_r1108310292